charts-other/sidam/charts/sidam-fr-am/templates/service.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-{{ .Chart.Name }}
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name }}-{{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}-{{ .Chart.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+spec:
+  selector:
+    app.kubernetes.io/name: {{ .Release.Name }}-{{ .Chart.Name }}
+  ports:
+    - name: ldap
+      port: 1389
+      targetPort: 1389
+    - name: web
+      port: 8080
+      targetPort: 8080
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: fr-am
+  labels:
```

Comment:
It's me not knowing Helm and chart (catching up!), but why we need to repeat it this? It's like the same as above but without name having placeholders...

Reply:
Duplicate will be removed as stated in other comment.